src/js/containers/dashboard/ActiveDashboardContainer.jsx

```diff
@@ -0,0 +1,74 @@
+/**
+ * ActiveDashboardContainer.jsx
+ * Created by Lizzie Salita 3/9/20
+ */
+
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { fetchSubmissions } from 'helpers/dashboardHelper';
+import ActiveDashboard from 'components/dashboard/ActiveDashboard';
+import NoResultsMessage from 'components/dashboard/NoResultsMessage';
+import LoadingMessage from 'components/dashboard/LoadingMessage';
+
+const propTypes = {
+    appliedFilters: PropTypes.object
+};
+
+const ActiveDashboardContainer = (props) => {
+    const [results, setResults] = useState([]);
+    const [error, setError] = useState(false);
+    const [loading, setLoading] = useState(true);
+
+    useEffect(() => {
+        setLoading(true);
+        const filters = props.appliedFilters.filters.active;
+        const payload = {
+            agency_codes: [filters.agency],
+            file_names: [filters.file]
+        };
+        if (filters.lastModified.start || filters.lastModified.end) {
+            payload.last_modified_range = {
+                start_date: filters.lastModified.start,
+                end_date: filters.lastModified.end
+            };
+        }
+        if (filters.submissionId) {
+            payload.submission_ids = [filters.submissionId];
+        }
+        if (filters.createdBy.id) {
+            payload.user_ids = [filters.createdBy.id];
+        }
+        fetchSubmissions({
+            filters: payload,
+            certified: 'false'
```

Comment:
I've never been fully clear on this, do we not want certified submissions in this list at all? They're technically also "active" in that they can be tweaked and edited (`certified` gets updated/certified submissions) so seeing them in their "current" state might be useful

Reply:
Great question - noting that we would change this to `certified: 'mixed'` if certified submissions are to be included